benchmarks/fast_serializable_check.py

```diff
@@ -28,13 +28,13 @@ def _bench(rounds, input_df, fast):
 def assess_speed(df_kind):
     rounds = 100
     input_df = input_test_data()[df_kind][0]
-    orig_config = anr._FAST_CHECK_DF_SERIALIZABLE
+    orig_config = anr.FAST_CHECK_DF_SERIALIZABLE
```

Comment:
Should this still point at anr? or at config?

Reply:
this is fine, as it will use the module level namespace which contains the import